robot/EDA/resources/ContactsPageObject.py

```diff
@@ -557,4 +575,27 @@ def Disable_enchanced_checkbox(self):
             "Enable Enhanced Preferred Phone Functionality setting has been cleared.\n" +
             "Saving changes.\n" +
             "Proper configuration is in place for testing 'Disable Preferred Phone enforcement'."
-        )
\ No newline at end of file
+        )
+
+    def Find_the_frame(self, loc):
```

Comment:
I might not be understanding how this works, but I don't see this used anywhere?

Reply:
Correct - but I have plans for it in next cycle.  It didn't make sense to take it out again.